views/about_me.erb

```diff
@@ -0,0 +1,28 @@
+<html>
+<head>
+  <title>About Me</title>
+</head>
+<header>
+  <h3><a href="index.html" />Home</a>   <a href="blog.html" />Blog</a>   <a href= "projects.html" />Projects</a></h3>
```

Comment:
Is there a way to accomplish the same layout but have each `a` tag on a separate line?